src/components/containers/menu/Menu.js

```diff
@@ -22,23 +28,34 @@ class Menu extends React.Component {
   };
 
   render() {
-    const { children, buttonContent, className } = this.props;
+    const {
+      children,
+      buttonContent,
+      className,
+      theme,
+      openButtonType
+    } = this.props;
 
     return (
-      <div className={className}>
-        <ToggleButton
-          handleOnClick={this.toggleMenu}
-          isPressed={this.state.isMenuOpen}
-        >
-          {buttonContent}
-        </ToggleButton>
-        <MenuPanel
-          isOpen={this.state.isMenuOpen}
-          closeFunction={this.toggleMenu}
-        >
-          {children}
-        </MenuPanel>
-      </div>
+      <ThemeProvider theme={theme}>
+        <div className={className}>
+          <StyledToggleButton
+            handleOnClick={this.toggleMenu}
+            isPressed={this.state.isMenuOpen}
+            styleType={openButtonType}
+            theme={theme}
```

Comment:
If the ToggleButton component was wrapped in a ThemeProvider (it ToggleButton.js, not here), you wouldn't need to pass the theme as a prop here. This ThemeProvider would automatically pass it down.  Same with the MenuPanel, though I did notice that it will need a default specified for its theme prop in MenuPanel.js.